events/api.py

```diff
@@ -1506,6 +1541,11 @@ def connected_organizations(self, connected_orgs, created_org):
             created_org.children.add(*conn_org)
 
     def create(self, validated_data):
+        # Add current user to admin users
+        if "admin_users" not in validated_data:
+            validated_data["admin_users"] = []
+        validated_data["admin_users"].append(self.user)
```

Comment:
I guess it is possible that after this the current user is duplicated in the list. Probably not an issue in practice, but a bit meh 🙂

Reply:
Fixed in https://github.com/City-of-Helsinki/linkedevents/pull/609/commits/f641a100ef900d0db69ef6f32661ff695389b917